### ISSUE TITLE
Missing Downloads

### ISSUE THREAD

**Issue by italofelipe** [FaisalUmair/udemy-downloader-gui#59]
Good morning, i'm trying to download a course, Udeler says Download Complete, but when I try to watch the videos, i see that they have only 300kb, and they don't even open. I love your app, it helps a lot, but plz, fix that!

**Comment by FaisalUmair** [comment]
Hi, Please try deleting the course folder in your downloads directory. There is a good chance that the download links are expired. 

I will surely fix it in the next release and make sure that the application takes care of the expired download links.

**Comment by duale1994** [comment]
Yes Bro FaisalUmair I have that same Problem and i have problem downloading the courses, it's too slow it takes to long to start the downloading  please fix it as soon as possible bro 
Thanks 
Sincerely
